src/Yaml.cs

```diff
@@ -17,18 +20,35 @@ public class Yaml {
         }
 
         public static async Task<T> LoadFromFileAsync<T> (string file) {
-            using (FileStream fs = File.OpenRead(file)) { 
+            using (FileStream fs = File.OpenRead(file)) {
                 return await LoadFromStreamAsync<T>(fs);
             }
         }
 
         public static T LoadFromString<T>(string content) {
-            var deserializer = 
+            var deserializer =
                 new DeserializerBuilder()
                 .WithNamingConvention(new CamelCaseNamingConvention())
                 .Build();
             var obj = deserializer.Deserialize<T>(content);
             return obj;
         }
+
+        public static string SaveToString<T>(T value)
+        {
+            var stringBuilder = new StringBuilder();
+            var writer = new StringWriter(stringBuilder);
+            var emitter = new Emitter(writer);
+
+            var serializer =
+                new SerializerBuilder()
+                .WithNamingConvention(new CamelCaseNamingConvention())
+                .BuildValueSerializer();
+            emitter.Emit(new StreamStart());
```

Comment:
necessary?

Reply:
The StreamStart thing? Yes, otherwise the emitter will throw an exception, unfortunately.